### PR TITLE
Update bottom toolbar UI

### LINKED ISSUE #11831: [Woo POS] Bottom toolbar UI updates

### PR DESCRIPTION
**Pull request by samiuelson**
Closes: #11831

### Description

This PR updates the UI of the bottom toolbar as per the design in Figma. It shows an explicit "Connect now" button and different icons for "disconnected" and "connected" states.

### Steps to reproduce

Focus on testing different states of reader connection and how they are reflected in the bottom toolbar.

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/4527432/8ab33c8a-2ea3-414b-a1a2-32cd7a4b8d11

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt [hunk 1]
```diff
@@ -40,23 +44,73 @@ fun WooPosBottomToolbar(
...
+
+@Composable
+private fun CardReaderStatus(
+    state: State<WooPosRootScreenState>,
+    onConnectToReaderClick: () -> Unit
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        when (state.value.cardReaderStatus) {
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt:84]: Just an idea - to not spend more than 5 mins on that. If to have each state emited fully in each time, then by wrapping it all in "AnimatedVisibility" we can have nice animation/transition between states

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt [hunk 2]
```diff
@@ -40,23 +44,73 @@ fun WooPosBottomToolbar(
...
+    }
+}
+
+@Composable
+private fun ExitPosButton(onClick: () -> Unit) {
+    TextButton(onClick = onClick) {
+        Icon(
+            painter = painterResource(id = R.drawable.woopos_ic_exit_pos),
+            contentDescription = null,
+            tint = Color.Unspecified
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt:65]: Shell we use `secondaryVariant` from theme here?

File: WooCommerce/src/main/res/drawable/woopos_ic_reader_connected.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (kidinov) [inline WooCommerce/src/main/res/drawable/woopos_ic_reader_connected.xml]: I think we can avoid having a vector for this and just draw a circle in compose

```
    Box(
        modifier = Modifier
            .size(12.dp)
            .background(color = Color.Blue, shape = CircleShape)
    )
```

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt [hunk 4]
```diff
@@ -40,23 +44,73 @@ fun WooPosBottomToolbar(
...
-                Text(
-                    text = stringResource(id = state.value.cardReaderStatus.title),
-                    color = MaterialTheme.colors.secondary,
-                    style = MaterialTheme.typography.button
+
+            WooPosCardReaderStatus.NotConnected -> {
+                Icon(
+                    painter = painterResource(id = R.drawable.woopos_ic_reader_disconnected),
+                    contentDescription = null,
+                    tint = Color.Unspecified
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt:93]: Maybe add the color to `WooPosTheme::CustomColors`?

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt [hunk 5]
```diff
@@ -67,7 +121,7 @@ fun PreviewWooPosBottomToolbar() {
     val state = remember {
         mutableStateOf(
             WooPosRootScreenState(
-                WooPosRootScreenState.WooPosCardReaderStatus.Unknown,
+                WooPosCardReaderStatus.NotConnected,
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt:135]: Maybe Previews for other states?

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt [hunk 6]
```diff
@@ -40,23 +44,73 @@ fun WooPosBottomToolbar(
...
+                    tint = Color.Unspecified
                 )
             }
-            TextButton(onClick = { onUIEvent(WooPosRootUIEvent.ConnectToAReaderClicked) }) {
-                Text(
-                    text = stringResource(id = state.value.cardReaderStatus.title),
-                    color = MaterialTheme.colors.secondary,
-                    style = MaterialTheme.typography.button
+
+            WooPosCardReaderStatus.NotConnected -> {
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt:89]: Shell we also add an icon for connecting state? I don't see this in the design, but if it's not there, I think circle with color between orange and green should do FFB8CC00

**Review by kidinov — APPROVED** [review]
LGTM! I left a couple of suggestions, please take a look. They are non blocking I believe

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt (hunk 4 above)
Comment (samiuelson) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt:93]: `Color.Unspecified` is used just to instruct compose not to apply any tint to the drawable. My thinking is that it's not a color, just an argument saying "don't apply the tint".

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt (hunk 6 above)
Comment (samiuelson) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt:89]: Done in [33634c9](https://github.com/woocommerce/woocommerce-android/pull/11846/commits/33634c9c2830b0e74e1385a9eeeaf54bb684f5ff)

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt (hunk 5 above)
Comment (samiuelson) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt:135]: Done in [33634c9](https://github.com/woocommerce/woocommerce-android/pull/11846/commits/33634c9c2830b0e74e1385a9eeeaf54bb684f5ff)

File: WooCommerce/src/main/res/drawable/woopos_ic_reader_connected.xml (hunk 3 above)
Comment (samiuelson) [reply WooCommerce/src/main/res/drawable/woopos_ic_reader_connected.xml]: Done in [33634c9](https://github.com/woocommerce/woocommerce-android/pull/11846/commits/33634c9c2830b0e74e1385a9eeeaf54bb684f5ff)

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt (hunk 4 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt:93]: But we must apply tint to prepare it for dark mode?

Comment (samiuelson) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/root/WooPosRootBottomToolbar.kt:93]: I'm not sure about that, maybe the original tint of the icon will fit both modes. Let's change it when we have some clarity about dark mode's design.

**Pull request merged**